src/content/1.7/modules/creation/enabling-auto-update.md

```diff
@@ -9,25 +9,34 @@ aliases:
 Enabling the Auto-Update
 ========================
 
-Since PrestaShop 1.5, it is possible to have your module auto-update:
-once a new version is available on Addons, PrestaShop suggests an
-"Update it!" button to the user. Clicking this button will trigger a
-series of methods, each leading closer to the latest version of your
-module.
+Since PrestaShop 1.5, it is possible to have your module auto-update: once a new version is available, PrestaShop suggests an "Update" button to the user. Clicking this button will trigger a series of methods, each leading closer to the latest version of your module.
 
-In order to bring auto-update support to your module, you need three
-main things:
+In order to bring auto-update support to your module, you need three main things:
 
--   Clearly indicate the module's version number in its constructor
-    method: `$this->version = '1.1'`;
--   Create an `/upgrade` sub-folder in the module's folder.
--   Add an auto-update PHP script for each new version.
+1. Clearly indicate the module's version number in its constructor method: `$this->version = '1.1'`;
+2. Create an `/upgrade` sub-folder in the module's folder.
+3. Add an auto-update PHP script for each new version.  
 
-For instance:
+Each script should containing a main method named `upgrade_module_x_y_z()`, where `x_y_z` corresponds to the version the module is being upgraded to. It receives a single parameter, an instance of your Module.
```

Comment:
```suggestion
Each script should contain a main method named `upgrade_module_x_y_z`, where `x_y_z` corresponds to the version the module is being upgraded to. It receives a single parameter, an instance of your Module.
```
Remove parenthesis of the method name, otherwise add the parameter.